test/UnitTest.cs

```diff
@@ -273,10 +277,12 @@ void TestTpcdsWithData()
         void TestTpchPlanOnly()
         {
             var files = Directory.GetFiles(@"../../../tpch", "*.sql");
-            string stats_fn = "../../../tpch/statistics/sf1";
+            //string stats_fn = "../../../tpch/statistics/sf1";
+            string stats_fn_1 = "../../../tpch/statistics/TPC-H_1G_Statistical_Data/sf1-withoutLineitem";
```

Comment:
is it possible to manually merge these two?

Reply:
Yes... I am in the process of merging the two statistics files now.

Reply:
To be filed and completed on another branch...